django_project/certification/views/certifying_organisation.py

```diff
@@ -217,6 +221,24 @@ def get_context_data(self, **kwargs):
         if project_slug:
             context['the_project'] = Project.objects.get(slug=project_slug)
             context['project'] = context['the_project']
+
+        if self.request.user.is_staff:
+            context['user_can_create'] = True
+            context['user_can_delete'] = True
+
+        if context['the_project'].certification_managers.filter(
```

Comment:
L222, `the_project` is created only it the condition is true. Here line L229, you use it, but it will raise a nexception of "unknown variable" if "the_project" doesn't exist. I'm not sure, but I think we should be able to remove the "if" line 221, is-it possible to have the project_slug?

Reply:
I think we can use the project_slug, let me test that.

Reply:
@Gustry I think we can go with it without the contion on L221.
I have updated.